withDom.js

```diff
@@ -1,3 +1,5 @@
+require('raf/polyfill');
```

Comment:
"compatible with shims" is still compatible; but I think it'd be better to avoid mutating the global in any situation inside `withDom`